test/e2e/kubernetes/kubernetes_test.go

```diff
@@ -190,14 +190,10 @@ var _ = Describe("Azure Container Cluster using the Kubernetes Orchestrator", fu
 				switch addonName {
 				case "blobfuse-flexvolume":
 					addonPods = []string{"blobfuse-flexvol-installer"}
-					addonNamespace = "flex"
```

Comment:
@jackfrancis In the addon array , can we change `blobfuse-flexvolume` to `blobfuse-flexvolume-installer` and `smb-flexvolume` to `smb-flexvolume-installer` ?

This way we wouldn't need unnecessary case statements and we. decrease the. cyclomatic complexiity.

Reply:
@tariq1890 that's good idea, I have renamed all flexvolume names to *-installer, cc @ritazh

Reply:
Sure :) That works. *-installer seems more suggestive of a DaemonSet